syntax = "proto3";

package fortuna.magicword;

message MagicWord {
    oneof magicWordResult {
        string word = 1;
        bool empty = 2;
    }
}

message MagicWordRequest {
    bool requested = 1;
}

service MagicWordService {
    rpc GetMagicWord(MagicWordRequest) returns (MagicWord);
}